src/environment.jl

```diff
@@ -172,8 +172,12 @@ or `false`.
 function has_cuda()
     flag = get(ENV, "JULIA_MPI_HAS_CUDA", nothing)
     if flag === nothing
-        # Only Open MPI provides a function to check CUDA support
-        @static if startswith(MPI_LIBRARY_VERSION, "Open MPI")
+        # Only OpenMPI provides a function to check CUDA support
+        # - Spectrum MPI is an OpenMPI, but IBM removed the functionality
+        #   check, therefore force true
+        @static if occursin(r".*Spectrum.*", MPI_LIBRARY_VERSION)
```

Comment:
Is this sufficiently specific to avoid picking up anything else? Should we make it match `"IBM Spectrum MPI"`?

Reply:
also, can't you just write
```julia
occursin("Spectrum", MPI_LIBRARY_VERSION)
```